src/tasks/job.ts

```diff
@@ -0,0 +1,49 @@
+type JobConstructorOpts = {
+  id: number
+  source: string
+  lang: string
+  timelimit?: number
+} 
+type RunJobConstructorOpts = JobConstructorOpts & {
```

Comment:
then this can be an another Interface that extends the Job interface

Reply:
done